packages/contract-addresses/CHANGELOG.json

```
@@ -1,4 +1,14 @@
 [
+    {
+        "version": "6.9.0",
+        "changes": [
+            {
+                "note": "Celo deployment",
+                "pr": 348
+            }
+        ],
+        "timestamp": 1634668033
```

this field is autogenerated
```suggestion
        ]
```